hitl/run.sh

```
@@ -0,0 +1,22 @@
+#!/usr/bin/bash -e
+
```

`set -eux` is usually a good choice. Otherwise commands will silently fail and not be printed.